source/class/cv/ui/manager/editor/Tree.js

```diff
@@ -2274,19 +2285,24 @@ qx.Class.define('cv.ui.manager.editor.Tree', {
               path: previewFile.getFullPath(),
               hash: 'ignore'
             },
-
             content,
-            () => {
-              qx.event.message.Bus.dispatchByName(previewFile.getBusTopic(), {
-                type: 'contentChanged',
-                file: previewFile,
-                data: content,
-                source: this
-              });
-
-              this.__modifiedPreviewElements.removeAll();
-              this.resetPreviewState();
-              previewFile.resetTemporary();
+            err => {
+              if (err) {
+                // disable preview, because the file could not be created
+                this.setShowPreview(false);
+                this.error(err);
+                cv.ui.manager.snackbar.Controller.error(this.tr('Disabling preview because the preview file could not be created.'));
```

Comment:
I didn't run the code but by looking at it: isn't the translation for this string missing?

Reply:
Yes you are right